src/styles/blocks/_nav.scss

```
@@ -0,0 +1,46 @@
+.nav {
+  &__list {
+  display: flex;
+}
+
+  &__item {
+  margin-right: 20px;
+}
+
+  &__item:last-child {
+  margin-right: 0;
+}
+
+  &__link {
+  display: block;
+  line-height: 60px;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+  color: black;
+
```

That's different level of nesting
```suggestion
  &__link {
    display: block;
    line-height: 60px;
    font-weight: 500;
    font-size: 12px;
    text-transform: uppercase;
    color: black;
  
```